src/addons/Portal/Portal.js

```diff
@@ -344,14 +347,15 @@ class Portal extends Component {
     if (!this.state.open) return
     debug('renderPortal()')
 
-    const { children, className, eventPool } = this.props
+    const { children, className, eventPool, style } = this.props
 
     this.mountPortal()
 
     // Server side rendering
     if (!isBrowser()) return null
 
     this.rootNode.className = className || ''
+    this.rootNode.style = style || ''
```

Comment:
I think this is susceptible to users passing style objects to the `<Portal style={...} />`.  Let me know if I'm misreading here.

Reply:
@levithomason correct. It is. Is there a reason you see that we should not allow that? If we do not want to allow that, is there another way you would prefer we do this?

Reply:
Hm, I think I was tripping out that day... Apologies, continue!